src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py

```diff
@@ -54,6 +54,7 @@ def upload_source_code(client,
         raise CLIError("Failed to get a SAS URL to upload context.")
 
     account_name, endpoint_suffix, container_name, blob_name, sas_token = get_blob_info(upload_url)
+    BlockBlobService = get_sdk(cmd.cli_ctx, ResourceType.DATA_STORAGE, 'blob#BlockBlobService')
```

Comment:
Can we import it statically like `from azure.storage.blob import BlockBlobService`?
What is the advantage of this approach?

Reply:
as what I described in description part